prover/cmd/prover/cmd/prove.go

```diff
@@ -42,11 +45,18 @@ func Prove(args ProverArgs) error {
 		if err := readRequest(args.Input, req); err != nil {
 			return fmt.Errorf("could not read the input file (%v): %w", args.Input, err)
 		}
+
 		// we use the large traces in 2 cases;
 		// 1. the user explicitly asked for it (args.Large)
 		// 2. the job contains the large suffix and we are a large machine (cfg.Execution.CanRunLarge)
 		large := args.Large || (strings.Contains(args.Input, "large") && cfg.Execution.CanRunFullLarge)
 
+		// check the arithmetization version used to generated the trace is contained in the prover request
```

Comment:
a typo: used to generated -> used to generate

Reply:
fixed, good catch :)